Makefile

```diff
@@ -53,12 +55,19 @@ TERRAFORM_DIR = terraform
 export BOOTSTRAP_ENV=terraform/bootstrap
 
 ifeq ($(ENV_NAME), dev)
-DISABLE_AUTOMATED_RECONCILIATION=true
+export DISABLE_AUTOMATED_RECONCILIATION=true
+export AWS_ACCOUNT_ID := $(AWS_ACCOUNT_ID_DEV)
+endif
+
+ifeq ($(ENV_NAME), test)
+export DISABLE_AUTOMATED_RECONCILIATION=true
```

Comment:
just confirming we want this disabled on test, since I think it was "false" originally?

Reply:
oh gosh! Thank you for catching that!